vm/src/format.rs

```diff
@@ -405,7 +406,7 @@ impl FormatSpec {
                     magnitude if magnitude.is_infinite() => Ok("inf".to_owned()),
                     // Using the Debug format here to prevent the automatic conversion of floats
                     // ending in .0 to their integer representation (e.g., 1.0 -> 1)
```

Comment:
You could probably remove this explanation, since it doesn't apply anymore.

Reply:
Good points!